### DISCUSSION TITLE
Help. Could not comiling Anura3d.  Configuration 'Release64 reuiqres compiler support for the x64 platform....

### DISCUSSION THREAD

**Discussion by HungTQ2023** [Anura3D/Anura3D_OpenSource#274] [Q&A]
[info@anura3D.com](mailto:info@anura3d.com)
![Screenshot 2024-11-02 050402](https://github.com/user-attachments/assets/cab47e21-1bd1-4b9b-98c1-8698170f960b)

How can I compile? Please support.

Thanks

**Comment by AVroonland** [comment]
I solved this problem by going to Project->Anura3D properties and changing the use compiler to IFX Intel Fortran Compiler
